polkadot/runtime/kusama/src/lib.rs

```diff
@@ -1712,6 +1713,19 @@ pub mod migrations {
 		}
 	}
 
+	pub struct ParachainsToUnlock;
+	impl Contains<ParaId> for ParachainsToUnlock {
+		fn contains(id: &ParaId) -> bool {
+			let id: u32 = (*id).into();
+			// ksuama parachains/parathreads that are locked and never produced block
+			match id {
+				2003 | 2077 | 2089 | 2111 | 2112 | 2127 | 2130 | 2224 | 2231 | 2233 | 2237 |
+				2256 | 2257 | 2258 | 2261 | 2268 | 2274 | 2275 => true,
```

Comment:
@SBalaguer I checked them, but can you please also again check them!

Reply:
With the condition of Parachains/Parathreads being locked and never producing one single block, I have 6 more paraIDs for Kusama, and one that does not apply:

`2003 | 2008 | 2018 | 2077 | 2089 | 2111 | 2112 | 2120 | 2126 | 2127 | 2130 | 2226 | 2227 | 2231 | 2237 | 2256 | 2257 | 2261 | 2268 | 2275`

The ones I'm adding are: 2,008, 2,018, 2,120, 2,126, 2,226, 2,227

The one that does not apply is paraID 2224 as it's unlocked.

Reply:
Confirmed @SBalaguer is correct

Reply:
I use `hrmp.hrmpWatermarks` to check if a parachain produced block. The lease swap is done by referendum.